kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -38,8 +38,10 @@ impl<'tcx> GotocCtx<'tcx> {
             Operand::Copy(place) | Operand::Move(place) =>
             // TODO: move is an opportunity to poison/nondet the original memory.
             {
-                let projection =
-                    unwrap_or_return_codegen_unimplemented!(self, self.codegen_place_stable(place));
+                let projection = unwrap_or_return_codegen_unimplemented!(
+                    self,
+                    self.codegen_place_stable(place, Location::none())
```

Comment:
Why is it right to keep this place without location?

Reply:
Constants in operands have a `span`, but places don't. I don't know whether this is an intentional design in stable MIR or just an omission. @celinval?

Reply:
Added a TODO comment.